.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+force_alphabetical_sort = True
```

Comment:
Resolved, had a remnant entry in a `setup.cfg`. The settings flag does not do what it says on the tin.

https://github.com/timothycrosley/isort/blob/d756957bad7bb3b5acaf3a2842c999f00b2fd83e/isort/settings.py#L138

Thank you, @jone.